src/terminal/osc.zig

```diff
@@ -3365,3 +3369,19 @@ test "OSC: kitty color protocol no key" {
     try testing.expect(cmd == .kitty_color_protocol);
     try testing.expectEqual(0, cmd.kitty_color_protocol.list.items.len);
 }
+
+// Note that this test doesn't really do much, as the parser seems to
+// be initialized differently when being run with `zig build test` vs a
+// normal build. In a normal build `p.command` appears as if it contained a
```

Comment:
Hm this is surprising and a bit unfortunate. Does this mean it's not possible to write unit tests that cover use of undefined values?

Reply:
Not that I can tell, at least not when it depends specifically on the undefined value. Which I guess makes sense because in a ReleaseFast version you'll just get whatever randomly happened to be in memory at that location.

Reply:
We should be able to eventually run our test exe under Valgrind and it'll catch undefined access.

Reply:
I've confirmed that Valgrind can in fact find all of this: #8304